test/evergreen.yml

```diff
@@ -1115,6 +1115,17 @@ tasks:
         vars:
           test_path: test/cppsuite/search_near_03
 
+  - name: cppsuite-bounded-cursor-perf-default
+    tags: ["pull_request"]
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "cppsuite test"
+        vars:
+          test_config_filename: configs/bounded_cursor_perf_default.txt
+          test_name: bounded_cursor_perf
```

Comment:
Do we want to get some statistics too for the default test as well? @luke-pearson

Reply:
No I don't think so.